File: Data Structures & Algorithms/counting-bits/submission-1.cpp
class Solution {
public:
    int calcOnes(int n){
        int q =0;
        int r =0;
        int res =0;
        while(n){
            n = n&(n-1);
            res++;
        }
        return res;
    }
    vector<int> countBits(int n) {
        vector<int>v {0};
        
        for(int i =1; i<=n;i++){
            v.push_back(calcOnes(i));
        }
        return v;
    }
};

File: Data Structures & Algorithms/is-subsequence/submission-1.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        m = len(s)
        n = len(t)
        i = 0
        j = 0
        if t == "":
            return False
        if s == "":
            return True
        while m:
            if j == len(t) -1 and m > 1:
                return False
            if s[i] == t[j]:
                i+=1
                j+=1
                m-=1
            else:
                j+=1
        return True
        